exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -58,60 +63,68 @@ private Adapter() {}
    * @param span the span to be converted
    * @return the Jaeger span
    */
-  static Model.Span toJaeger(Span span) {
+  static Model.Span toJaeger(SpanData span) {
     Model.Span.Builder target = Model.Span.newBuilder();
 
-    target.setTraceId(span.getTraceId());
-    target.setSpanId(span.getSpanId());
+    target.setTraceId(TraceProtoUtils.toProtoTraceId(span.getTraceId()));
+    target.setSpanId(TraceProtoUtils.toProtoSpanId(span.getSpanId()));
```

Comment:
I'm wondering about the need to create a new class with these two methods instead of creating them in this class as was done with things like `toProtoTimestamp`

Reply:
I moved that class over from the sdk. I'm fine moving the methods into the Adapter. Doesn't matter too much to me, either way.

Reply:
ended up moving this class over to the `otoproto` package for future usage.